typings/glamorous.d.ts

```diff
@@ -62,7 +76,7 @@ export function withTheme<ExternalProps, Theme>(
   component: React.ComponentClass<ExternalProps & { theme: Theme }>
 ): React.ComponentClass<ExternalProps>
 
-export function withTheme<ExternalProps, Theme>(
+  export function withTheme<ExternalProps, Theme>(
```

Comment:
Should investigate using Omit to make this easier similar to styled-components. 

https://github.com/styled-components/styled-components/pull/982/files